docs/ldml/tr35.md

```diff
@@ -1725,6 +1727,37 @@ There are certain invariants that must always be true:
 4. There must never be cycles, such as: X parent of Y ... parent of X.
 5. Following the inheritance path, using parentLocale where available and otherwise truncating the locale, must always lead eventually to the root locale.
 
+#### 4.1.4 <a name="Region_Priority_Inheritance" href="#Region_Priority_Inheritance">Region-Priority Inheritance</a>
+
+Certain data may be more appropriate to store with the region as the primary key instead of language. This is often needed for regional user preferences, such as week info, calendar system, and measurement system.
+
+The default search chain for region-priority inheritance removes the language subtag before the region subtag, and it restores the variant subtag after doing so, as follows:
```

Comment:
We have never done reattachment of variants before, for normal item lookup. The variant is really more closely attached to the base language than to the region, so I think the "US_someVariant" step should be removed. 

----
Longer term, I think we need a rewrite of inheritance to have clean separation, and treat resource bundle lookup and item lookup in separate sections, and use distinct terminology in each, and always use the same terminology.

Reply:
For this to be usable, we have to be precise about exactly which items use region inheritance, and which don't. I'd propose that we say that this applies to all item in rgScope (found in rgScope.xml). Note that some of the attributes that are named "region" take regions or locales, eg

<!ATTLIST hours regions NMTOKENS #REQUIRED >
    <!--@MATCH:set/or/validity/region||validity/locale-->

Reply:
Thanks for the reference to rgScope. How does this look? I added a link to the existing section on rgScope. Do we need to say anything about attributes named "region" taking regions or locales?